R/geo_objects.R

```diff
@@ -0,0 +1,56 @@
+# Contains functions to parse the objects described here: 
+# https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/geo
+
+bounding_box <- function(x) {
+  empty <- data.frame(long = NA, lat = NA, type = NA)
+  if(is.null(x) || length(x) == 1 && is.na(x)) {
```

Comment:
```suggestion
  if (is.null(x) || (length(x) == 1 && is.na(x))) {
```

Reply:
Another check would be `is.null(x) || identical(x, NA)` (assuming that it's always a logical NA)

Reply:
I corrected the style and changed to use the identical check. It didn't break any test, so I think it is safe to assume it will be always a logical NA.